front/app/components/admin/FormResults/FormResultsQuestion/SentimentQuestion/components/GroupedSentimentScore/transformGroupedAnswerUsableArray.test.ts

```diff
@@ -0,0 +1,103 @@
+import { GroupedAnswer } from 'api/survey_results/types';
+
+import { transformGroupedAnswerUsableArray } from './utils';
+
+describe('transformGroupedAnswerUsableArray', () => {
```

Comment:
Had not realised that we had to do this transformation on the front-end - wonder why we didn't just return it from the backend in the correct format?

Reply:
I'm not sure if we had a spec written up beforehand or not - but I assumed that this followed the same pattern for other "Grouped Answers" for other question types? Is that not the case though? :thinking: